tle/cogs/acd_ai.py

```diff
@@ -24,19 +24,33 @@
 class ACD_AI(commands.Cog):
     def __init__(self, bot):
         self.bot = bot
+        
+        '''ids of text channels which will be used for opening chat threads with ai'''
```

Comment:
avoid using ''' for single line comments. 
let's add checks to lint later; in the upcoming MR